streamly.cabal

```diff
@@ -96,6 +96,8 @@ extra-source-files:
     test/Streamly/Test/Data/*.hs
     test/Streamly/Test/Data/Array/CommonImports.hs
     test/Streamly/Test/Data/Array/Common.hs
+    test/Streamly/Test/Data/Array/CommonCoreImports.hs
+    test/Streamly/Test/Data/Array/CommonCore.hs
```

Comment:
Can we just change the existing CommonImports.hs and Common.hs instead of introducing copies og those?

Reply:
effort is more than benefits.

Reply:
CommonCoreImports.hs and CommonCore.hs has been removed.